src/plugins/not/not.js

```diff
@@ -0,0 +1,22 @@
+// .not(elems)
+// Delete all of the nodes that equals elems
+u.prototype.not = function(elems){
+  if(typeof elems == 'object'){
```

Comment:
Please see the "big" comment at the end of the discussion, it's relevant for the rest of the code:

> https://github.com/umbrellajs/umbrella/pull/15